src/Microsoft.Dnx.Runtime.Abstractions/Constants.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Dnx.Runtime
```

Comment:
Ok, but we do already expose the values. These aren't RIDs, they are the values we expose publicly on `IRuntimeEnvironment`